src/workerd/api/node/tests/tls-nodejs-tcp-server.js

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+// This file is used as a sidecar for the net-nodejs-test tests.
+// It creates 2 TCP servers to act as a source of truth for the node:net tests.
```

Comment:
This says it creates two tcp servers but there's only one created in this script?

Reply:
Outdated comment... I'll remove it.